models/contacts.js

```diff
@@ -1,19 +1,81 @@
-// const fs = require('fs/promises')
+import fs from "fs/promises";
+import { nanoid } from "nanoid";
+import path from "path";
 
-const listContacts = async () => {}
+const contactsPath = path.resolve("models", "contacts.json");
 
-const getContactById = async (contactId) => {}
+export const listContacts = async () => {
+  try {
+    const contacts = fs.readFile(contactsPath);
+    const parsedContacts = JSON.parse(await contacts);
 
-const removeContact = async (contactId) => {}
+    return parsedContacts;
+  } catch (error) {
+    console.log(error);
+  }
+};
 
-const addContact = async (body) => {}
+export const getContactById = async (contactId) => {
+  try {
+    const contacts = fs.readFile(contactsPath);
```

Comment:
const contacts = `await` fs.readFile(contactsPath); can you place `await` like it is in my example and remove this one under?

Reply:
I changed position of await in every function.